.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,5 @@
 <!--- Provide a brief summary of the issue in the title above -->
+<!--- We'd kindly ask you to open any issues related to the Gradle Android plugin or Android studio on the Google forum at `https://source.android.com/source/report-bugs` -->
```

Comment:
❌ Wording:

Android studio -> Android Studio
Google forum ->  Android Issue Tracker

⭕️ I think we can also add (assuming that the more precise the instructions are the more likely people follow them):
...in the components _App Development > Gradle_ or _App Development > Android Studio > Gradle IDE_.
(see component selection here https://issuetracker.google.com/issues/new)